source/guides/getting-started.md

```diff
@@ -955,9 +956,9 @@ require 'spec_helper'
 require_relative '../../../../apps/web/views/books/new'
 
 class NewBookParams < Hanami::Action::Params
-  param :book do
-    param :title, presence: true
-    param :author, presence: true
+  params :book do
+    required(:title).filled(:str?)
+    required(:author).filled(:str?)
   end
```

Comment:
Disregard if this is a work in progress and not ready to be reviewed, but with the new validations it looks like this needs to be:

```ruby
params do
  required(:book).schema do
    ...
  end
end
```